frontend/src/pages/pipelines/global/modelCustomization/baseModelSection/BaseModelSection.tsx

```diff
@@ -63,18 +63,19 @@ const BaseModelSection: React.FC<BaseModelSectionProps> = ({
         </FlexItem>
       </Flex>
       <FormGroup label="Model registry name" fieldId={`${FIELD_ID_PREFIX}-registryName`}>
-        {registryName ?? '-'}
+        <div data-testid="base-registry-name">{registryName ?? '-'}</div>
```

Comment:
Just checking if we can directly add `data-testid` to FormGroup or if we can use label value as a selector here instead of having a extra div
The same applies to the following

Reply:
I've tried but with Fromgroup it also fetches the label in it.